app/views/catalog/_universal_viewer_default.html.erb

```diff
@@ -1,3 +1,3 @@
 <div class="uv__overlay<%= " condensed" if current_exhibit&.condensed_viewer %>" onClick="style.pointerEvents='none'">
-  <iframe allowfullscreen="true" src="<%= universal_viewer_url %>"></iframe>
+  <iframe title="Viewer for IIIF Manifest" allowfullscreen="true" src="<%= universal_viewer_url %>"></iframe>
```

Comment:
What do we think about this title? OK? Something else?

Reply:
"Digital Content"?

Reply:
Udpated.